node/forest_libp2p/src/service.rs

```diff
@@ -849,6 +849,7 @@ pub fn get_keypair(path: &Path) -> Option<Keypair> {
         Ok(mut vec) => match ed25519::Keypair::decode(&mut vec) {
             Ok(kp) => {
                 info!("Recovered libp2p keypair from {:?}", &path);
+                #[allow(deprecated)] // FIXME: use: into_ed25519
```

Comment:
Can we do it in this PR? If not, let's include a tracking issue in the comment. Wild FIXMEs tend to get forgotten.

Reply:
Added in this PR.